src/ralph/assets/utils.py

```diff
@@ -14,13 +14,20 @@ def move_parents_models(from_obj, to_obj):
     Args:
         from_obj: From Django model object
         to_obj: To Django modelo object
+        exclude_copy_fields: Remove field from object dict
 
     Example:
         >> instance = DataCenterAsset.objects.get(pk=1)
         >> back_office_asset = BackOfficeAsset()
         >> move_parents_models(instance, back_office_asset)
     """
-    to_obj.__dict__.update(from_obj.__dict__)
+    from_dict = from_obj.__dict__
+    if exclude_copy_fields:
+        for field in exclude_copy_fields:
+            from_dict.pop(field, None)
```

Comment:
what do you think about it:
```
for field_name in exclude_copy_fields:
    field = to_obj._meta.get_field(field_name)
    from_dict.pop(field.name, None)
    from_dict.pop(field.attname, None)  # this should be <field>_id in general, but it could be custom
```